src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala

```diff
@@ -0,0 +1,173 @@
+    case object G_callvalue extends Key
+    case object G_callstipend extends Key
+    case object G_newaccount extends Key
+    case object G_exp extends Key
+    case object G_expbyte extends Key
+    case object G_memory extends Key
+    case object G_txcreate extends Key
+    case object G_txdatazero extends Key
+    case object G_txdatanonzero extends Key
+    case object G_transaction extends Key
+    case object G_log extends Key
+    case object G_logdata extends Key
+    case object G_logtopic extends Key
+    case object G_sha3 extends Key
+    case object G_sha3word extends Key
+    case object G_copy extends Key
+    case object G_blockhash extends Key
+    case object G_extcodesize extends Key
+    case object G_extcodecopy_base extends Key
+    case object G_selfdestruct_to_new_account extends Key
+  }
+
+  val FrontierFeeSchedule: FeeSchedule = FeeSchedule(Map(
+      Key.G_zero -> UInt256(0),
+      Key.G_base -> UInt256(2),
+      Key.G_verylow -> UInt256(3),
+      Key.G_low -> UInt256(5),
+      Key.G_mid -> UInt256(8),
+      Key.G_high -> UInt256(10),
+      Key.G_extcode -> UInt256(700),
+      Key.G_balance -> UInt256(20),
+      Key.G_sload -> UInt256(50),
+      Key.G_jumpdest -> UInt256(1),
+      Key.G_sset -> UInt256(20000),
+      Key.G_sreset -> UInt256(5000),
+      Key.R_sclear -> UInt256(15000),
+      Key.R_selfdestruct -> UInt256(24000),
+      Key.G_selfdestruct -> UInt256(0),
+      Key.G_create -> UInt256(32000),
+      Key.G_codedeposit -> UInt256(200),
+      Key.G_call -> UInt256(40),
+      Key.G_callvalue -> UInt256(9000),
+      Key.G_callstipend -> UInt256(2300),
+      Key.G_newaccount -> UInt256(25000),
+      Key.G_exp -> UInt256(10),
+      Key.G_expbyte -> UInt256(10),
+      Key.G_memory -> UInt256(3),
+      Key.G_txcreate -> UInt256(21000),
+      Key.G_txdatazero -> UInt256(4),
+      Key.G_txdatanonzero -> UInt256(68),
+      Key.G_transaction -> UInt256(21000),
+      Key.G_log -> UInt256(375),
+      Key.G_logdata -> UInt256(8),
+      Key.G_logtopic -> UInt256(375),
+      Key.G_sha3 -> UInt256(30),
+      Key.G_sha3word -> UInt256(6),
+      Key.G_copy -> UInt256(3),
+      Key.G_blockhash -> UInt256(20),
+      Key.G_extcodesize -> UInt256(20),
+      Key.G_extcodecopy_base -> UInt256(20),
+      Key.G_selfdestruct_to_new_account -> UInt256(0)))
+
+  val HomesteadFeeSchedule: FeeSchedule =
+    FrontierFeeSchedule.copy(values = FrontierFeeSchedule.values ++ Map(
+      Key.G_txcreate -> UInt256(53000)))
+
+  val PostEIP150FeeSchedule: FeeSchedule =
+    HomesteadFeeSchedule.copy(values = HomesteadFeeSchedule.values ++ Map(
+      Key.G_sload -> UInt256(200),
+      Key.G_call -> UInt256(700),
+      Key.G_balance -> UInt256(400),
+      Key.G_selfdestruct -> UInt256(5000),
+      Key.G_extcodesize -> UInt256(700),
+      Key.G_extcodecopy_base -> UInt256(700),
+      Key.G_selfdestruct_to_new_account -> UInt256(25000),
+      Key.G_expbyte -> UInt256(10)))
```

Comment:
@LukasGasior1 this value seems to be the same as Frontier config, has it changed? I haven't found a new definition in EIP150 neither....am I doing something wrong?